tequilapi/endpoints/payout.go

```diff
@@ -34,7 +34,13 @@ type payoutInfo struct {
 	// in Ethereum address format
 	// required: true
 	// example: 0x000000000000000000000000000000000000000a
-	EthAddress   string `json:"ethAddress"`
+	EthAddress string `json:"ethAddress"`
```

Comment:
We are using a different style for JSON tags.
`ethAddress` - camalCase
`referral_code` - snake_case